Update lco-ingester dependency version

irasnyd opened the pull request:
The lco-ingester library has been updated to version 2.1.14, which
itself contains an update to one of its dependencies,
opentsdb-python-metrics, which publishes metrics data on the standard
HTTP port 80 by default, rather than using a custom port number.

This behavior is also now configurable, rather than being hard-coded
into the library.

With this change, the LCO IT staff will be able to migrate the OpenTSDB
server to our newer hardware platform.

Commit:
Update lco-ingester dependency version

The lco-ingester library has been updated to version 2.1.14, which
itself contains an update to one of its dependencies,
opentsdb-python-metrics, which publishes metrics data on the standard
HTTP port 80 by default, rather than using a custom port number.

This behavior is also now configurable, rather than being hard-coded
into the library.

With this change, the LCO IT staff will be able to migrate the OpenTSDB
server to our newer hardware platform.

irasnyd commented:
Note that I am almost positive that `pip_requirements.txt` is unused.

Previous to this commit, the `pip_requirements.txt` file contained a requirement of `lco_ingester==2.1.5` which is mutually incompatible with `setup.py` which specifies `lco_ingester>=2.1.7`. (Obviously `lco_ingester==2.1.5` does not satisfy `lco_ingester>=2.1.7`. Therefore one of these files must not be used. Let's remove the unused one?)

I'll let you guys actually merge and deploy this. There is no urgent rush, however this will really help the IT Team significantly in the very near term. We would appreciate if this can get merged and deployed within the next week or so.

The Ingester Library updates are already deployed as part of the Shipper.

mgdaily commented:
> Note that I am almost positive that `pip_requirements.txt` is unused.
> 
> Previous to this commit, the `pip_requirements.txt` file contained a requirement of `lco_ingester==2.1.5` which is mutually incompatible with `setup.py` which specifies `lco_ingester>=2.1.7`. (Obviously `lco_ingester==2.1.5` does not satisfy `lco_ingester>=2.1.7`. Therefore one of these files must not be used. Let's remove the unused one?)

Yeah, good find. I'm thinking the master build is probably broken for that reason. The Dockerfile does use `pip_requirements.txt` during the build process, so I am hesitant to remove it without Curtis here. 

Jenkins seems to build the docker image on master happily, but I cannot build master locally, due to a similar dependency issue:

```
error: opentsdb-python-metrics 0.1.8 is installed but opentsdb-python-metrics>=0.2.0 is required by {'lco-ingester'}
The command '/bin/sh -c python /nres/code/setup.py install' returned a non-zero code: 1
```

Curtis does not currently have master deployed, however. He is on the `fix/queue_backup` branch, which does have `lco-ingester==2.1.11` in `pip_requirements.txt`, and `setup.py` specifies `lco_ingester>=2.1.7`. So the currently deployed branch has valid dependency constraints.

In any case, this isn't your problem to deal with. II have changed the base branch of the PR to point to the currently deployed branch, and I will try to get Curtis to review it soon. He is PTO through Wednesday, so worst-case we deploy Thursday.

mgdaily committed:
Merge branch 'fix/queue_backup' into upgrade-ingester-library

mgdaily committed:
Pin version of astroquery.

When merging changes from the base branch, I accidentally removed the pinning on astroquery. This commit re-instates that.
